Overtone.Game/Windows/MainScene.fs

```diff
@@ -0,0 +1,54 @@
+﻿namespace Overtone.Game.Windows
+
+open JetBrains.Lifetimes
+open Microsoft.Xna.Framework
+open Microsoft.Xna.Framework.Graphics
+open Microsoft.Xna.Framework.Input
+
+open Overtone.Game
+open Overtone.Game.Config
+open Overtone.Game.Constants
+open Overtone.Game.Input
+
+type MainScene(lifetime: Lifetime,
+               device: GraphicsDevice,
+               textureManager: Textures.Manager,
+               config: WindowsConfiguration) =
+    let mutable sceneId = Scenes.MainMenu // "Starting state" of windows.txt
+
+    let mutable controls = config.GetControlsArray sceneId
+    let loadControl element = Controls.Load(lifetime, textureManager, element)
+
+    // let title = Control(
+    //     textureManager.LoadTexture(lifetime, Shapes.TitleScreen.Id, Shapes.TitleScreen.TitleFrame),
+    //     None,
+    //     Rectangle(0, -41, 640, 480),
+    // )
```

Comment:
Ow, that (an absence of the game title) is a regression. Let's get it back somehow? At least with a check for the current scene for now?

Though I believe that we may as well choose to extract some micro-controllers for scenes into separate types/functions, but that we may do later.

Reply:
I can put it back in the if and create an issue

Reply:
f1be88bf906be275748dcd3e52f7f138e07d6ed4

Reply:
cf https://github.com/ForNeVeR/overtone/issues/48